text/source/components/subsystems/power_supply.rst

```diff
@@ -82,7 +82,7 @@ We can augment our system model to include an additional load (that
 comes online after some delay):
 
 .. image:: /ModelicaByExample/Subsystems/PowerSupply/Examples/AdditionalLoad.*
-   :width: 100%
+   :width: 80%
```

Comment:
Actually, this kind of thing is best handled with `figclass` if they are figures.  I changed a bunch a few commits ago.  Basically, you can add extra classes for the figure and then control width from the CSS.  This is useful because you can use media queries there to decide sizes based on screen size.  I'll accept these for now.  but ideally these should be managed via CSS.

Reply:
This is all well and good but does not help for the pdf. So please keep
these and also keep them maintained.